src/microcosm.js

```diff
@@ -195,35 +195,57 @@ class Microcosm extends Emitter implements Domain {
 
   recall(action: ?Action) {
     if (action && action.id in this.snapshots) {
-      return this.snapshots[action.id]
+      return this.snapshots[action.id].next
     }
 
     return this.initial
   }
 
   /**
+   * Get the state prior to a given action, but include any upstream
+   * updates from parent Microcosms.
+   */
+  rebase(action: Action) {
+    let state = this.recall(action.parent)
+
+    if (this.parent) {
+      state = merge(state, this.parent.recall(action))
+    }
+
+    return state
+  }
```

Comment:
Devtools rely on `recall` to get the state snapshot for an action, so I'm afraid to touch it. We should come up with a public API for getting the snapshot for an action, use that in the dev tools, and revisit this.